licensing/src/main/java/io/crate/license/Cryptos.java

```diff
@@ -30,17 +32,49 @@
 import java.nio.charset.StandardCharsets;
 import java.security.InvalidKeyException;
 import java.security.Key;
+import java.security.KeyFactory;
+import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.security.spec.X509EncodedKeySpec;
 
 final class Cryptos {
 
     private static final String PASSPHRASE = "crate_passphrase";
     private static final String CIPHER_ALGORITHM = "AES";
     private static final Key AES_KEY_SPEC = new SecretKeySpec(PASSPHRASE.getBytes(StandardCharsets.UTF_8), "AES");
+    private static final String KEY_ALGORITHM = "RSA";
 
     private Cryptos() {
     }
 
+    /**
+     * Read public key file content
+     */
+    static PublicKey readPublicKey(byte[] fileContents) {
```

Comment:
`read` is imho miss-leading, it's not reading anything as the file contents are already provided?

Reply:
done